pkg/router/router.go

```diff
@@ -200,6 +200,7 @@ func registerWeb(container *restful.Container) {
 		if webResourcesStaticPattern.Match([]byte(r.URL.Path)) && !webResourcesStaticExcludePattern.Match([]byte(r.URL.Path)) {
 			// Static resources are immutable and have a content hash in their URL
 			w.Header().Set("Cache-Control", "public, max-age=31536000, immutable")
+			w.Header().Set("X-Frame-Options", "deny")
```

Comment:
As mentioned on the working group call this needs to be outside the conditional since the conditional only matches the JS, CSS, and image resources.

Ideally we'd only add the `X-Frame-Options` response header on requests for the landing page (`/index.html` or just `/`), but shouldn't have any negative impact just adding it to everything if that proves awkward.